pilot/pkg/networking/util/util.go

```diff
@@ -61,19 +61,21 @@ func ConvertAddressToCidr(addr string) *core.CidrRange {
 		return nil
 	}
 
-	cidr := &core.CidrRange{
-		AddressPrefix: addr,
-		PrefixLen: &types.UInt32Value{
-			Value: 32,
-		},
-	}
-
+	var cidr *core.CidrRange
 	if strings.Contains(addr, "/") {
 		parts := strings.Split(addr, "/")
 		cidr.AddressPrefix = parts[0]
```

Comment:
This will cause a null pointer error :). Thats the reason I allocated apriori

Reply:
get it, thx